trendmicro_apex/plugin.spec.yaml

```diff
@@ -126,3 +718,546 @@ actions:
+          - CM
+          - CMEF
+          - OSF_COMMAND
+          - OSF_QUERY
+          - OSF_NOTIFY
+          - OSF_LOG
+          - MDR_ATTACK_DISCOVERY
+          - OSF_SYS_CALL
+      content_id:
+        title: Content ID
+        description: Indicates the location of the dataset
+        type: string
+        required: false
+        example: 8
+      limit:
+        title: Limit
+        description: Retrieves the top n results from the servers
+        type: string
+        required: true
+        example: "1"
+    output:
+      Data:
+        title: Data
+        description: Contains the data returned by the specific API
+        type: api_data
+        required: true
+      FeatureCtrl:
+        title: "Feature CTRL"
+        description: The Apex Central deployment model
+        type: FeatureCtrl
+        required: false
+      Meta:
+        title: "Meta"
+        description: Indicates the response status, including the result, error code, and error message
+        type: Meta
+        required: false
+      PermissionCtrl:
+        title: "Permission CTRL"
+        description: Indicates the permissions assigned to the logged-on user account for accessing Apex Central menu items and features
+        type: PermissionCtrl
+        required: false
+      SystemCtrl:
+        title: "System CTRL"
+        description: Indicates the suspicious object distribution role of the Apex Central server
+        type: SystemCtrl
+        required: false
+  terminate_process:
+    title: Terminate Process
+    description: Terminates the processes specified on Security Agent endpoints
+    input:
+      agent_guid:
+        title: Agent GUID
+        description: The key is the serverGuid, and the value is a list of agentGuid strings of the endpoints managed by the target server
+        type: object
+        required: false
+        example: '{"2EBEC86D-3FEB-4666-9CA6-B80AB1E193E6": ["B7F34FF9-9DEE-45F4-9F24-DC5116C79D52"]}'
+      filter:
+        title: Filter
+        description: The filter paramter is useful for terminating the amount of agents without specifying the agentGuid
+        type: "[]filter"
+        required: false
+        example: '[{"type": 1, "value": "a"}]'
+      server_guid:
+        title: Server GUID
+        description: GUID of servers which manage the endpoints specified in agentGuid
+        type: "[]string"
+        required: false
+        example: '["2EBEC86D-3FEB-4666-9CA6-B80AB1E193E6"]'
+      suspicious_object_name:
+        title: Suspicious Object Name
+        description: File name of the object to terminate
+        type: string
+        required: false
+        example: "file.txt"
+      termination_info_list:
+        title: Termination Info List
+        description: Container for terminationInfoList objects
+        type: "[]termination_info"
+        required: true
+        example: '[{"name": 101, "value": "2FF40C5ED6E5A3BBC68A10F2966F347463E326AD"}]'
+      task_type:
```

Comment:
@mgorny-morecode I think we should remove this option because users will always want to terminate processes on the "Endpoint sensor"  and just hard code the value.